modules/ocf/manifests/systemd/timer.pp

```diff
@@ -0,0 +1,27 @@
+# Creates two systemd units: a timer and service bearing the same
+# name, whereby the service will be activated once the timer elapses.
+define ocf::systemd::timer(
+  $timer_content = undef,
+  $service_content = undef,
+  $timer_source = undef,
+  $service_source = undef,
+) {
+  # ocf::systemd::service will handle multiple timer sources
+  if $service_content and $service_source {
+    fail('You may not supply both content and source parameters to ocf::systemd::timer')
+  }
+  elsif $service_content == undef and $service_source == undef {
+    fail('You must supply either the content or source parameter to ocf::systemd::timer')
+  }
+  file { "/etc/systemd/system/${title}.service":
```

Comment:
Nope, I modified ocf::systemd::service to take a custom suffix. The timer is created there and the service is started; once that elapses the service file with the same name is activated. 

It essentially does the same thing as this https://gist.github.com/kkuehler/f0664592519900ff4fd5df93508d86bd.

I've actually been debating between using the current design and the one I linked above. Opinions are welcome.

Reply:
Ah, good point, I saw the suffix but forgot about that, I think that works fine then.

Reply:
I still don't know if it's the best idea to use `ocf::systemd::service` to manage timers. I'm worried it could lead to confusion because in this case we use `ocf::systemd::service` to manage the timer unit and not the actual service unit. I'm not sure that generalizing `ocf::systemd::service` is thus the right way to go here. Does anyone else have thoughts?